lib/systems/examples.nix

```diff
@@ -0,0 +1,130 @@
+# These can be passed to nixpkgs as either the `localSystem` or
```

Comment:
The the stdenv needs to be *cross*-bootstrapped is just an artifact of history. The platform description itself doesn't care about the provenence of the stdenv.

More practically :), I wanted to remove all mention of "cross" here as a friendly reminder that the `localSystem` and `crossSystem` syntax is now the same.